src/components/AppHeader/AppHeader.tsx

```diff
@@ -52,8 +53,15 @@ const useStyles = makeStyles({
 
 export const AppHeader = () => {
   const classes = useStyles();
-  // const navigate = useNavigate();
   const userContext = useContext(UserContext);
+  const roles = useRoles();
+
+  const myRoles = roles.data?.filter(
+    (item) => Number(item.user.Id) === userContext.user.Id
+  );
+
+  const isAdmin =
+    myRoles?.filter((item) => item.Title === "Admin").length === 1;
```

Comment:
If the hook I suggest above returns an array of roles -- you can use the includes method to determine if it includes "Admin"